### PR TITLE
Better way to define paths for CORS endpoints. r=fabrice

### PR DESCRIPTION
**Pull request by ferjm**

### CODE REVIEW

File: src/service_router.rs [hunk 1]
```diff
@@ -21,8 +21,8 @@ impl CORS {
     // Endpoints containing a variable path part can use '*' like in:
     // &["bar", "*"] for a URL like https://foo.com/bar/123
```
Comment (fabricedesre) [inline src/service_router.rs]: This comment needs an update right?

Comment (ferjm) [inline src/service_router.rs]: Yes. I just changed it. Thanks!

**Comment by fabricedesre** [comment]
r=me

**Pull request merged**